### PR TITLE
Updates lesson for Swift 2.X

### PR DESCRIPTION
**Pull request by sloik**

### CODE REVIEW

File: LiveLessons4/Lesson4_5_functionalProgramming.playground/contents.swift [hunk 1]
```diff
@@ -20,19 +20,6 @@ roundedUpSavings
 let doubleDigits = transactions.filter { $0 >= 10.0 }
 doubleDigits
 
-// currying
```
Comment (sloik) [inline LiveLessons4/Lesson4_5_functionalProgramming.playground/contents.swift:23]: Currying will be removed in Swift 3 :)

Comment (SixFiveSoftware) [inline LiveLessons4/Lesson4_5_functionalProgramming.playground/contents.swift:23]: I don't think currying will be removed completely, rather, the syntax as it currently is to create a curried function will be removed. You would still be able to declare a function which can accept a parameter and return a function expecting the next parameter, as outlined in Detailed Design here (https://github.com/apple/swift-evolution/blob/master/proposals/0002-remove-currying.md). Please remove this commit from the PR and I will review again.

**Comment by sloik** [comment]
Updated :)

**Comment by SixFiveSoftware** [comment]
Great, thank you! I'll merge this in.

**Pull request merged**